### ISSUE TITLE
Rename directory named `aux`

### ISSUE THREAD

**Issue by mpilgrem** [quchen/prettyprinter#252]
It is not possible to `git clone` this repository on Windows because it has used `aux` as the name of a directory. `AUX` is the name of a reserved device in Windows. See: https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file

**Comment by sjakobi** [comment]
@mpilgrem: We have several symbolic links to the [`version-compatibility-macros.h`](https://github.com/quchen/prettyprinter/pull/253/files#diff-75bba398053d7094f341af10852b83895de9c262d3840884d26d10cc4343c645) file within that `aux` directory. Will these symbolic links also give trouble on Windows?

**Comment by mpilgrem** [comment]
@sjakobi, I don't think so. Unless Developer Mode is turned on, a user needs Adminstrator privileges to _create_ symlinks on Windows - however, I think all users can _follow_ (and delete) created symlinks.

The MSYS2 project has some material on symlinks, including a short primer on native Windows symlinks: https://www.msys2.org/docs/symlinks/
